src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/filter/filter.tsx

```diff
@@ -81,91 +64,30 @@ class Filter extends React.Component<Props, State> {
     return checkedQuantity;
   };
 
-  filterOptionsFirstStep = (options: Array<Option>, option: Option) => {
-    const filteredOptions = options.map(type => {
-      if (isEqual(type, option)) {
+  filterOptionsFirstStep = <T extends Option>(options: Array<T>, option: T): Array<T> => {
+    return options.map(opt => {
+      if (isEqual(opt, option)) {
```

Comment:
Can we please rename one of these? It's kind of hard to read and understand when there are two variables with similar names.